CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Attributor Changelog
 ============================
 
+next
+------
+
+* `Model` is now a subclass of `Hash`... expect things to break
```

Comment:
this should contain a few sub-bullets about changes and implications of the change.